setup.py

```diff
@@ -8,7 +9,8 @@
 
 setuptools.setup(
     name="icepyx",
-    version="0.3.2",
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
```

Comment:
Hi @lsetiawan, just a quick comment if you don't mind. Is it possible to use [setuptools_scm](https://github.com/pypa/setuptools_scm) instead of [versioneer](https://github.com/python-versioneer/python-versioneer) to set the version? This will keep the `icepyx` top directory cleaner. See https://github.com/psf/black/pull/1008 for an example PR.

```suggestion
    use_scm_version={"local_scheme": "node-and-date"},
```

Reply:
I haven't used setuptools_scm before, but I'll try it out. Seems a lot slimmer! Thanks for providing this suggestion and example :smile:

Reply:
Looks good! Less files to worry about down the track :smile: